src/components/UI/Button.jsx

```diff
@@ -0,0 +1,30 @@
+import { Link } from "react-router-dom";
+
+import PropTypes from "prop-types";
+
+const Button = ({ destination, styles, children, onClick }) => {
+  return (
+    <>
+      {destination ? (
+        <Link to={destination}>
+          <button className={styles} onClick={onClick}>
+            {children}
+          </button>
+        </Link>
+      ) : (
+        <button className={styles} onClick={onClick}>
+          {children}
+        </button>
+      )}
+    </>
+  );
+};
+
+export default Button;
+
+Button.propTypes = {
+  destination: PropTypes.string,
+  styles: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+  onClick: PropTypes.func,
+};
```

Comment:
```suggestion
Button.propTypes = {
  styles: PropTypes.string.isRequired,
  children: PropTypes.node.isRequired,
  destination: PropTypes.string,
  onClick: PropTypes.func,
};
```
필수인 props들을 최상단에 위치시키는게 더 좋을 것 같은데, 어떻게 생각하시나요?

Reply:
좋습니다..! 가독성에 더 좋네요. 반영하도록 할게요!

Reply:
동의합니다~ 다른 분의 코멘트더라도 이렇게 의견 남겨놓겠습니다! 모두의 의견이 체크된 상태라는게 인지되면 좋을 것 같아서요 😀